### PR TITLE
Converted multiple links to GitHub, upto line number 550

### PR DESCRIPTION
**Pull request by parthnagdev**
Replaced links up to line number 550 in LIST.md file for those projects which had a GitHub repository. 
Some projects have their source code in other git services like Bitbucket/Source Forge/GitLab, for such projects and for those whom I could not find any source code repositories, I made no changes to the link. Especially, for some reason, many projects marked with "![c]" seem to not be an Open-Source Software. Those projects which have a GitHub project page but not a specific GitHub repo, for such projects I also did not make any changes to the link EXCEPT for project at line number 115, doing this prevents skipping of the project in the main website and this is for you to review. I will be completing rest of the replacements by tomorrow hopefully. Please do let me know if I broke something or if did not follow a best practice of making a commit/editing someone else's code. Thanks for bearing me.

### CODE REVIEW

File: src/public/LIST.md [hunk 1]
```diff
@@ -190,42 +190,42 @@ _Efficient and specific data structures._
...
+- [Apache Calcite](https://github.com/apache/calcite) - Dynamic data management framework. It contains many of the pieces that comprise a typical database management system.
+- [Apache Drill](https://github.com/apache/drill) - Distributed, schema on-the-fly, ANSI SQL query engine for Big Data exploration.
+- [Apache Phoenix](https://github.com/apache/phoenix) - High-performance relational database layer over HBase for low-latency applications.
 - [ArangoDB](https://github.com/arangodb/arangodb-java-driver) - ArangoDB Java driver.
 - [Chronicle Map](https://github.com/OpenHFT/Chronicle-Map) - Efficient, in-memory (opt. persisted to disk), off-heap key-value store.
-- [Debezium](https://debezium.io) - Low latency data streaming platform for change data capture.
-- [druid](https://druid.apache.org) - High-performance, column-oriented, distributed data store.
+- [Debezium](https://github.com/debezium/debezium) - Low latency data streaming platform for change data capture.
+- [druid](hhttps://github.com/apache/druid/) - High-performance, column-oriented, distributed data store.
```
Comment (ResamVi) [inline src/public/LIST.md]: Small mistake here: `hhttps`

File: src/public/LIST.md [hunk 2]
```diff
@@ -301,12 +301,12 @@ _Tools that handle the distribution of applications in native formats._
 - [Artipie](https://github.com/artipie/artipie) - Binary artifact management toolkit which hosts them on the file system or S3.
 - [Boxfuse ![c]](https://boxfuse.com) - Deployment of JVM applications to AWS using the principles of immutable infrastructure.
 - [Capsule](https://github.com/puniverse/capsule) - Simple and powerful packaging and deployment. A fat JAR on steroids, or a "Docker for Java" that supports JVM-optimized containers.
-- [Central Repository](https://search.maven.org) - Largest binary component repository available as a free service to the open-source community. Default used by Apache Maven, and available in all other build tools.
+- [Central Repository](https://github.com/sonatype-nexus-community/search-maven-org/) - Largest binary component repository available as a free service to the open-source community. Default used by Apache Maven, and available in all other build tools.
```
Comment (ResamVi) [inline src/public/LIST.md]: Leave the trailing `/` out otherwise it won't work

File: src/public/LIST.md [hunk 3]
```diff
@@ -497,54 +497,54 @@ _Libraries for serializing and deserializing JSON to and from Java objects._
...
-- [Graylog](https://www.graylog.org) - Open-source aggregator suited for extended role and permission management. (GPL-3.0-only)
-- [Kibana](https://www.elastic.co/kibana) - Analyzes and visualizes log files. Some features require payment.
-- [Logback](http://logback.qos.ch) - Robust logging library with interesting configuration options via Groovy.
+- [Graylog](https://github.com/Graylog2/graylog2-server) - Open-source aggregator suited for extended role and permission management. (GPL-3.0-only)
+- [Kibana](https://github.com/elastic/kibana) - Analyzes and visualizes log files. Some features require payment.
+- [Logback](https://github.com/qos-ch/logback) - Robust logging library with interesting configuration options via Groovy.
 - [Logbook](https://github.com/zalando/logbook) - Extensible, open-source library for HTTP request and response logging.
-- [Logstash](https://www.elastic.co/logstash) - Tool for managing log files.
+- [Logstash](https://github.com/elastic/logstash - Tool for managing log files.
```
Comment (ResamVi) [inline src/public/LIST.md]: Missing `)`

**Review by ResamVi — COMMENTED** [review]
Very good effort. If you fix the comments I placed we can merge this

**Comment by parthnagdev** [comment]
Fixed.

**Pull request merged**